app/javascript/products.vue

```diff
@@ -195,12 +195,18 @@ export default {
       window.scrollTo(0, 0)
     },
     newUrl(pageNumber) {
-      const params = new URL(location.href).searchParams
+      const params = new URL(location.origin).searchParams
       if (pageNumber !== 1) params.set('page', pageNumber)
       if (this.params.target) params.set('target', this.params.target)
       if (this.params.checker_id)
         params.set('checker_id', this.params.checker_id)
-      return `${location.pathname}?${params}`
+      if (
+        params.get('page') ||
+        params.get('target') ||
+        params.get('checker_id')
+      )
+        return `${location.pathname}?${params}`
+      else return location.pathname
```

Comment:
```suggestion
      return location.pathname
```
すぐ上のifがtrueの時はreturnでブロックを抜けるので、ここのelseはない方がすっきり書けると思います。
上にある185行目の`getPageValueFromParameter()`内でもelseのない書き方になってました。

Reply:
コメントありがとうございます🙏
おっしゃる通りです！勉強になりました🙇‍♀️
修正いたしましたので、ご確認よろしくお願い致します！